cilium-cli/cli/connectivity.go

```diff
@@ -178,6 +178,9 @@ func newCmdConnectivityTest(hooks api.Hooks) *cobra.Command {
 	cmd.Flags().StringSliceVar(&params.ExpectedXFRMErrors, "expected-xfrm-errors", defaults.ExpectedXFRMErrors, "List of expected XFRM errors")
 	cmd.Flags().MarkHidden("expected-xfrm-errors")
 
+	cmd.Flags().StringSliceVar(&params.LogCheckLevels, "log-check-levels", defaults.LogCheckLevels, "Log levels to check for in error logs")
```

Comment:
nit: `Log levels to check for in log messages` ? (as it's actually not an `error log`)

Reply:
Good point, fixed.